site/pages/join-us/style.css

```diff
@@ -11,3 +12,58 @@
 .noWrap {
   white-space: nowrap;
 }
+
+.headerContainer > div {
```

Comment:
Think we should avoid descendant selectors in our CSS unless it's immediately clear what element it applies to or there is really no other selector we can use.